packages/chain-controller/README.md

```diff
@@ -0,0 +1,15 @@
+# `@metamask/chain-controller`
```

Comment:
I see that there are a lot of parts to this controller package with the existence of `ChainController`, `SnapHandlerSender`, `SnapChainProviderClient`, etc.

It would be good to have a diagram and/or short explanation in the README about how the relationship between each of these components.

Reply:
That's a good idea, I'll see what I can do! Thanks